### PR TITLE
Add php-stubs/wp-cli-stubs

### PR DESCRIPTION
**Pull request by szepeviktor**
From https://wordpress.slack.com/archives/C02RP4T41/p1600270987051800

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Third-party packages that are commonly useful.
 - [Yoast/wp-cli-faker](https://github.com/Yoast/wp-cli-faker) - Generate fake WordPress and WooCommerce content for testing purposes.
 - [aaemnnosttv/wp-cli-login-command](https://github.com/aaemnnosttv/wp-cli-login-command) - Login to WordPress with secure passwordless magic links.
 - [schlessera/wp-cli-psysh](https://github.com/schlessera/wp-cli-psysh) - Replace WP-CLI shell standard REPL with [PsySH](http://psysh.org/).
+- [php-stubs/wp-cli-stubs](https://github.com/php-stubs/wp-cli-stubs) WP-CLI function and class declaration stubs for static analysis.
```
Comment (schlessera) [inline readme.md]:
```suggestion
- [php-stubs/wp-cli-stubs](https://github.com/php-stubs/wp-cli-stubs) - WP-CLI function and class declaration stubs for static analysis.
```

**Review by schlessera — CHANGES_REQUESTED** [review]

**Comment by schlessera** [comment]
Thanks!

File: readme.md (hunk 1 above)
Comment (szepeviktor) [reply readme.md]: dashy-dash!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**